app/src/main/kotlin/com/wire/android/di/CoreLogicModule.kt

```diff
@@ -175,6 +176,11 @@ class CoreLogicModule {
         )
     }
 
+    @Singleton
```

Comment:
i guess there is no need for the Singleton annotation now since it is already a singilton in core logic